src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -30,11 +30,14 @@
     WPI_TalonSRX controlPanel = new WPI_TalonSRX(43);
 
     @Override
-    public DifferentialDriveMap getDriveMap() {
+    public DriveKinematics getDriveMap() {
         // 1/12.27 is the gear ratio multiplied by the circumfrence of the wheel
+        // multiplied by a correcion factor found by comparing distance traveled to
+        // encoder values
         final int averageCount = 15;
-        final double distancePerRev = (1.0 / 12.27) * (6.0 * Math.PI);
-        return new DifferentialDriveMap() {
+        final double distancePerRev = Units.inchesToMeters((1.0 / 12.27) * (6.0 * Math.PI)) * 0.9533562075675308;
+        // TODO find real trackwidth in Meters
```

Comment:
Lets actually implement that function for `FrancoisMap` instead of having a `TODO` here.